packages/base/src/SVGIconRegistry.js

```diff
@@ -1,7 +1,7 @@
 const registry = new Map();
 
-const registerIcon = (name, d) => {
-	registry.set(name, { name, d });
+const registerIcon = (name, d, acc) => {
+	registry.set(name, { name, d, acc });
```

Comment:
name is redundant here as it is the key, but it's not from your change so you can decide to fix it or not :)

Reply:
Done :)